front/components/assistant/markdown/CiteBlock.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect } from "react";
```

Comment:
Assuming it's a copy/paste?

Reply:
whole code is just moved, no change here